Support/Testing/Travis/install.py

```diff
@@ -75,7 +79,10 @@
     packages.append('realpath')
 
 if len(packages) > 0:
-    if "CentOS Linux" in platform.linux_distribution():
+    if travis_os == 'osx':
+        check_call('brew install "%s" || true' % '" "'.join(packages), shell=True)
```

Comment:
I know why you're doing this `|| true` thing with brew but I wish there was a better way to take care of this :/
Can you add a tiny comment to explain why so that next person to read this code gets it?